packages/email/template-components/template-document-self-signed.tsx

```diff
@@ -49,7 +49,7 @@ export const TemplateDocumentSelfSigned = ({
         <Text className="mx-auto mb-6 mt-1 max-w-[80%] text-center text-base text-slate-400">
           Create a{' '}
           <Link
-            href={`${process.env.NEXT_PUBLIC_WEBAPP_URL}/signup`}
+            href={`${process.env.NEXT_PUBLIC_WEBAPP_URL ?? 'http://localhost:3000'}/signup`}
```

Comment:
don't you think this will be neater doing it in a variable?

Reply:
of cuz it has nothing to do with your PR, but could be a good first issue

Reply:
Hi Dun-sin!

Thanks for pointing that out, it made me realise that this path is also required somewhere else in this file.

I'll update it as part of this PR, thanks!

Reply:
I noticed it's a universal thing in some other files